### ISSUE TITLE
Please update to Packet Tracer 8.2 and publish to Flathub

### ISSUE THREAD

**Issue by Anifyuli** [losuler/com.cisco.PacketTracer#13]
Hello, Mister. Packet Tracer reached to 8.2 version. I can install Packet Tracer in Fedora but it's very complicated. Please read my issue. Thanks

**Comment by losuler** [comment]
This had already been updated to Packet Tracer 8.2, and I confirmed was working, almost 2 months ago https://github.com/losuler/com.cisco.PacketTracer/commit/288832f6a33d9b17ef6560b21ac1d445c53f4843. I did notice I hadn't updated the [`README.md`](https://github.com/losuler/com.cisco.PacketTracer/blob/master/README.md) where I mention the version, but I've changed that now.

I'm running Fedora and it only need takes 4 commands to get this running. I realise it's not ideal, but it shouldn't be difficult. If you're having any problems though, please let me know.

As for adding to this to Flathub, see my comments about this in an older issue https://github.com/losuler/com.cisco.PacketTracer/issues/3#issuecomment-979843421. As far as I'm aware, this still hasn't changed.

**Comment by Anifyuli** [comment]
So, Packet Tracer can't publish on Flatpak, Mister? No chance for it? 
I think is simpler than manual extracting, copying to /opt and /usr, symlink and don't forget refresh icon cache

**Comment by losuler** [comment]
I'm not sure what you're talking about, none of the steps that you mentioned (manual extracting, copying to /opt and /usr, symlink, etc) are needed to use this Flatpak. You only need to build the Flatpak with `flatpak-builder`.

As for this being released on Flathub, I'm not aware of it being possible because Flatpak would need a mechanism to pass the credentials required to download the `.deb`.

**Comment by Anifyuli** [comment]
No, I'm talking about how to install Packet Tracer in Fedora without Flatpak builder. I think your project (this) simpler than my previous steps to get Packet Tracer in my Fedora

**Comment by koaledu** [comment]
This would be incredibly useful, it's really a chore to need building your own package with the .deb on Cisco's website.

**Comment by losuler** [comment]
> This would be incredibly useful, it's really a chore to need building your own package with the .deb on Cisco's website.

This isn't something I can resolve. If you want this to be possible, you need to bring it up with the Flatpak/Flathub devs.

**Comment by kRHYME7** [comment]
@losuler  Just want to say thank you!

I'll also add this here  for ref. 
https://discourse.flathub.org/t/request-cisco-packet-tracer/5297/3
https://github.com/flathub/flathub/pull/4229
